routers/web/org/home.go

```diff
@@ -101,11 +103,11 @@ func Home(ctx *context.Context) {
 	}
 
 	var (
-		repos []*repo_model.Repository
-		count int64
-		err   error
+		unpinnedrepos []*repo_model.Repository
```

Comment:
As mentioned already, I think it would be a good idea to keep the GitHub approach to still display everything as it keeps the code clearer and looks just as good.